src/ica_api.c

```diff
@@ -1242,6 +1242,28 @@ int ica_ecdsa_verify(ica_adapter_handle_t adapter_handle,
 	return rc;
 }
 
+int ica_ec_key_get_public_key(ICA_EC_KEY *key, unsigned char *q, unsigned int *q_len)
```

Comment:
can key parameter be qualified by const? it seems to be read only in this context.

Reply:
Yes! Will do ...